vms/event/tests/test_api.py

```diff
@@ -0,0 +1,70 @@
+# third party
+import json
+from rest_framework.test import APITestCase
+
+# Django
+from django.urls import reverse
+from django.test.testcases import TestCase
+
+# local Django
+from shift.utils import create_event_with_details
+
+
+class ApiForVolaViewTestCase(APITestCase, TestCase):
+    """
+    Contains Tests for Vola API
+    """
+
+    def setUp(self):
+        event = ['event', '2015-02-05', '2015-05-05']
+        self.event_1 = create_event_with_details(event)
+        self.event_1.city = 'city'
+        self.event_1.state = 'state'
+        self.event_1.country = 'country'
+        self.event_1.address = 'address'
+        self.event_1.venue = 'venue'
+        self.event_1.save()
+
+        event = ['eventq', '2050-02-05', '2050-05-05']
+        self.event_2 = create_event_with_details(event)
+        self.event_2.city = 'cityq'
+        self.event_2.state = 'stateq'
+        self.event_2.country = 'countryq'
+        self.event_2.address = 'addressq'
+        self.event_2.venue = 'venueq'
+        self.event_2.save()
+
+    def test_api_for_vms_get(self):
+        """Test GET request to provide data of all events"""
+        url = reverse('event:vola_api')
+        response = self.client.get(url)
+        self.assertEqual(json.loads(response.content), [{u'event_name': u'event',
```

Comment:
I think we can define this dict at class level rather than hardcoding here, might be used in future.